library/src/main/java/nl/dionsegijn/steppertouch/StepperTouch.kt

```diff
@@ -51,6 +51,7 @@ class StepperTouch : FrameLayout, OnStepCallback {
     private var stepperTextColor = R.color.stepper_text
     private var stepperButtonColor = R.color.stepper_button
     private var stepperTextSize = 20
+    private var justAddAvailable = false
```

Comment:
also I think `allowNegativeStepper = false` is not bad, because I think `hideNegativeStepper` is weird because yes we hide it and additionally we don't allow use it.